jlm/rvsdg/gamma.hpp

```diff
@@ -43,21 +50,80 @@ class GammaOperation final : public StructuralOperation
   virtual bool
   operator==(const Operation & other) const noexcept override;
 
+  /**
+   * \brief Returns the type of pattern matching content produced in a branch
+   */
+  std::shared_ptr<const Type>
+  GetMatchContentType(std::size_t alternative) const;
+
 private:
   size_t nalternatives_;
```

Comment:
Maybe you can change that name as well once you are at it.

Reply:
renamed